### PR TITLE
Update LICENSE.txt

### PR DESCRIPTION
**Pull request by Sylfrena**

### CODE REVIEW

**Review by metalmatze — APPROVED** [review]

**Comment by kakkoyun** [comment]
@maxbrunet was tracking a patch to fix it. Maybe we should wait for comment.

**Comment by maxbrunet** [comment]
Yes, we need to add the flag `--use-current-year` in `.pre-commit-config.yaml` instead. We can leave an old date in `LICENSE.txt`

See https://github.com/Lucas-C/pre-commit-hooks/releases/tag/v1.4.1

File: scripts/LICENSE.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2022 The Parca Authors
+Copyright 2023 The Parca Authors
```
Comment (Sylfrena) [inline scripts/LICENSE.txt:1]:
```suggestion
Copyright 2022 The Parca Authors
```

**Comment by Sylfrena** [comment]
@maxbrunet is this what you expected? It seems doing this will require the change to _all_ files...would you like me to accept the suggestions and update the PR?

**Pull request merged**